schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -81,6 +83,8 @@
               {
                 "oneOf": [
                   { "$ref": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.KeyVault.json#/resourceDefinitions/vaults" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.Cdn.json#/resourceDefinitions/profiles" },
```

Comment:
Here is the structure:
Microsoft.Cdn/Profiles
Microsoft.Cdn/Profiles/Endpoints
Microsoft.Cdn/Profiles/Endpoints/Origins
Microsoft.Cdn/Profiles/Endpoints/CustomDomains
So both profiles and endpoints have child resources?